pkg/products/grafana/reconciler.go

```diff
@@ -60,26 +42,17 @@ func (r *Reconciler) GetPreflightObject(_ string) k8sclient.Object {
 }
 
 func (r *Reconciler) VerifyVersion(installation *integreatlyv1alpha1.RHMI) bool {
-	return version.VerifyProductAndOperatorVersion(
-		installation.Status.Stages[integreatlyv1alpha1.InstallStage].Products[integreatlyv1alpha1.ProductGrafana],
-		string(integreatlyv1alpha1.VersionGrafana),
-		string(integreatlyv1alpha1.OperatorVersionGrafana),
-	)
+	return true
 }
 
 func NewReconciler(configManager config.ConfigReadWriter, installation *integreatlyv1alpha1.RHMI, mpm marketplace.MarketplaceInterface, recorder record.EventRecorder, logger l.Logger, productDeclaration *marketplace.ProductDeclaration) (*Reconciler, error) {
-	if productDeclaration == nil {
-		return nil, fmt.Errorf("no product declaration found for grafana")
-	}
-
-	ns := installation.Spec.NamespacePrefix + defaultInstallationNamespace + "-operator"
 	productConfig, err := configManager.ReadGrafana()
 	if err != nil {
 		return nil, fmt.Errorf("could not retrieve grafana config: %w", err)
 	}
 
-	productConfig.SetNamespace(ns)
-	productConfig.SetOperatorNamespace(productConfig.GetNamespace())
+	nsPrefix := installation.Spec.NamespacePrefix
+	productConfig.SetNamespace(nsPrefix + "customer-monitoring")
```

Comment:
in the alert reconciler can we not use the getProductNamespace instead of creating a new variable with namespace?

Reply:
Updated. Thanks very much for comment